src/compressor/QatAccel.cc

```diff
@@ -13,29 +13,34 @@
  */
 #include <qatzip.h>
 #include "QatAccel.h"
+#include "common/ceph_context.h"
+#include "common/common_init.h"
+#include "common/debug.h"
+#include "common/dout.h"
+#include "common/errno.h"
+
+// -----------------------------------------------------------------------------
+#define dout_context g_ceph_context
+#define dout_subsys ceph_subsys_compressor
+#undef dout_prefix
+#define dout_prefix _prefix(_dout)
+
+static std::ostream& _prefix(std::ostream* _dout)
+{
+  return *_dout << "QatAccel: ";
+}
+// -----------------------------------------------------------------------------
 
 void QzSessionDeleter::operator() (struct QzSession_S *session) {
-  if (NULL != session->internal) {
-    qzTeardownSession(session);
-    qzClose(session);
-  }
+  qzTeardownSession(session);
   delete session;
 }
 
 /* Estimate data expansion after decompression */
 static const unsigned int expansion_ratio[] = {5, 20, 50, 100, 200};
 
-QatAccel::QatAccel() {
-  session.reset(new struct QzSession_S);
-  memset(session.get(), 0, sizeof(struct QzSession_S));
-}
-
-QatAccel::~QatAccel() {}
-
-bool QatAccel::init(const std::string &alg) {
-  QzSessionParams_T params = {(QzHuffmanHdr_T)0,};
+static bool getQzParams(const std::string &alg, QzSessionParams_T &params) {
```

Comment:
we don't use camelCase naming convention. see https://github.com/ceph/ceph/blob/master/CodingStyle for more details.

Reply:
done